brambling/utils/model_tables.py

```diff
@@ -615,7 +638,7 @@ def get_field_val(self, obj, key):
             field = "spaces"
 
         if date_str:
-            if obj.get_eventhousing():
+            if obj.get_eventhousing() and obj.providing_housing:
```

Comment:
I've been playing with this a little and it seems like something else is happening in that other block where if it doesn't happen I get errors like this:

`'Order' object has no attribute 'hosting_spaces_20160731'`